src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -721,16 +721,20 @@ private static int calculateTagColumn(Matcher javadocTagMatcher,
         final DetailAST throwsAST = ast
                 .findFirstToken(TokenTypes.LITERAL_THROWS);
         if (throwsAST != null) {
-            DetailAST child = throwsAST.getFirstChild();
-            while (child != null) {
-                if (child.getType() == TokenTypes.IDENT
-                        || child.getType() == TokenTypes.DOT) {
+            DetailAST exception = throwsAST.getFirstChild();
```

Comment:
Please don't rename variables in scope of one update, it is hard to see what was actually changed/added.

Reply:
done, recover to original name